test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -82,6 +72,69 @@ public void CanChangeEngineFromScriptEditorDropDown()
             Assert.AreEqual(true, cPython3MenuItem.IsChecked);
         }
 
+        /// <summary>
+        /// This test checks if its changing the engine via 
+        /// dropdown selector inside the script editor executes the most up to date code.
+        /// </summary>
+        [Test]
+        public void ChangingDropdownEngineSavesCode()
+        {
+            // Arrange
+            var engineChange = PythonNodeModels.PythonEngineVersion.CPython3;
+
+            Open(@"core\python\python.dyn");
+
+            var nodeView = NodeViewWithGuid("3bcad14e-d086-4278-9e08-ed2759ef92f3");
+            var nodeModel = nodeView.ViewModel.NodeModel as PythonNodeModels.PythonNodeBase;
+            Assert.NotNull(nodeModel);
+            var scriptWindow = EditPythonCode(nodeView,View);
+
+            var engineSelectorComboBox = FindEditorDropDown(scriptWindow);
+
+            // Act
+
+            //modify code in editor
+            Assert.AreEqual("ok",(nodeModel as PythonNode).Script);
+            SetTextEditorText(scriptWindow, "OUT = 100");
+            //modify engine
+            engineSelectorComboBox.SelectedItem = engineChange;
+
+            //assert model code is updated.
+            Assert.AreEqual("OUT = 100", (nodeModel as PythonNode).Script);
+
```

Comment:
Can you also assert that the value of the node is 100? Also, a test name like `ChangingDropdownEngineSavesCodeBeforeRunning` would better express what we want to achieve here.